appcues/src/main/java/com/appcues/analytics/ExperienceLifecycleTracker.kt

```diff
@@ -38,12 +39,12 @@ internal class ExperienceLifecycleTracker(
     private val storage: Storage by inject()
     private val appcuesCoroutineScope: AppcuesCoroutineScope by inject()
 
-    fun start(stateMachine: StateMachine, dispatcher: CoroutineDispatcher = Dispatchers.IO) {
-        appcuesCoroutineScope.launch(dispatcher) { stateMachine.observeState() }
+    fun start(stateMachine: StateMachine, onDismissed: (Experience) -> Unit, dispatcher: CoroutineDispatcher = Dispatchers.IO) {
```

Comment:
maybe call this callback `onExperienceEnded` to more generally capture the state transition to `EndingExperience` - which could be dismissal or completion